### DISCUSSION TITLE
20240711 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#102] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

Issues:

- https://github.com/torrust/torrust-tracker/issues/948
- https://github.com/torrust/torrust-tracker/issues/950
- https://github.com/torrust/torrust-tracker/issues/324

I've reopened issue #324, and I've confirmed that even with the new hyper version (1.4.0), the problem is still there; there is no default timeout to close the connection.

I've created a [relase plan](https://github.com/torrust/torrust-tracker/wiki/Releases#v300) together with @cgbosse.

### Torrust Index 

Issues:

- https://github.com/torrust/torrust-index/issues/654
- Update dependencies
- Review security advisory (pending feedback from @da2ce7)

Comments on issues:

- https://github.com/torrust/torrust-index/issues/567

I've created a [relase plan](https://github.com/torrust/torrust-tracker/wiki/Releases#v300) together with @cgbosse.

### Torrust Index GUI

I've created a [relase plan](https://github.com/torrust/torrust-index-gui/wiki/Releases#v300) together with @cgbosse.

### Torrust Demo

- Update demo following changes in configuration files.

### Torrust Website

- Meeting with @grmbyrn to start working on the new website.

### Torrust Rebranding

Review proposals received from the designer for the new logo.

### Learning

Starting reading a new Rust book: [Rust in Action](https://www.manning.com/books/rust-in-action).

### Eng. Manager

One-on-ones with @grmbyrn and @mario-nt

**Comment by mario-nt** [comment]
### Index
	
- https://github.com/torrust/torrust-index/pull/662

### Reviews


### Training/Research


- Are you using Object-Oriented Programming? | Vaughn Vernon lecture

### Other

- One on One with Constantin and Jose

### New issues

**Comment by grmbyrn** [comment]
## Torrust website
* Research best ways to structure v2 of website
* Read through SvelteKit docs
* One on one with Jose to discuss the best way forward with v2
* Examine Figma and decide how to proceed with implementing it in v2
* Begin implementation of v2

## Learning
* Revise Python

**Comment by mario-nt** [comment]
### Index
	

- https://github.com/torrust/torrust-index/pull/662


### Training/Research


- Are you using Object-Oriented Programming? | Vaughn Vernon lecture

### Other

- One on One with Constantin and Jose
